### ISSUE TITLE
Possibility to wrap GridBox

### ISSUE THREAD

**Issue by surgura** [dattn/dnd-grid#23]
Is possible to wrap a GridBox component in your own component? For example:
``` html
<template>
<dnd-grid-box class="container" :boxId="boxId" dragSelector="div.header">
  <div class="header">Header</div>
  <div class="content">Content</div>
</dnd-grid-box>
</template>
(...)
```

So that it can be used as such:
``` html
<div id="app">
  <dnd-grid-container :layout.sync="layoutJson">
    <container boxId="box1Id"/>
  </dnd-grid-container>
</div>
```

This Container can then of course be whatever you want.

This way of doing it throws an error. I assume this is because GridBox tries to reference GridContainer, but finds the custom Container class instead.
![image](https://user-images.githubusercontent.com/10170564/36689604-240daee0-1b30-11e8-9bae-e4c7e9b3b8cb.png)

**Comment by dattn** [comment]
It there a reason why you can't it do this way ?
``` html
<div id="app">
  <dnd-grid-container :layout.sync="layoutJson">
    <dnd-grid-box class="container" :boxId="boxId" dragSelector="div.header">
      <your-custom-component />
    </dnd-grid-box>
  </dnd-grid-container>
</div>
```

**Comment by surgura** [comment]
What I want to achieve is somehting like this:
``` html
<custom-container>
  <custom-box/>
  <custom-box/>
  <custom-box/>
</custom-container>
```

Where both have styling and `custom-box` can have `slots` for e.g. header and content.

I think this looks a lot nicer than
``` html
<custom-container-layout>
  <dnd-grid-container>
    <dnd-grid-box>
      <custom-box-content/>
    </dnd-grid-box>
    <dnd-grid-box>
      <custom-box-content/>
    </dnd-grid-box>
    <dnd-grid-box>
      <custom-box-content/>
    </dnd-grid-box>
  </dnd-grid-container>
</custom-container-layout>

Also, when you have multiple grids this leads to a lot of repetition.

**Comment by dattn** [comment]
Extending the Box and/or the Container container should work.
``` js
let CustomBox = {
  extends: DndGridBox,
  ...
}
```

**Comment by surgura** [comment]
👍 I will give that a try later on today.

**Comment by dattn** [comment]
This commit should fix you issue: https://github.com/dattn/dnd-grid/commit/704b37e66b046f9285daaf6f614365dd65f8ad3c
I will release a new version soon

**Comment by dattn** [comment]
Fixed in v0.0.13 release
